charts/fybrik/templates/manager-deployment.yaml

```diff
@@ -175,7 +175,7 @@ spec:
         {{- if .Values.manager.solver.image }}
         - name: solver-volume
           emptyDir:
-            sizeLimit: {{ .Values.manager.solver.dirSizeLimit }}
+            sizeLimit: {{ .Values.manager.solver.solverVolumeSizeLimit }}
```

Comment:
suggest to rename to volumeSizeLimit because it is already under solver

Reply:
Changed to dirSizeLimit